### ISSUE TITLE
Installation fails

### ISSUE THREAD

**Issue by DoctorOak** [ember-animation/liquid-fire#515]
When installing via `ember install liquid-fire` I get the following error message:

```D:\wamp64\www\__PROJECT__\node_modules\liquid-fire\index.js:83
      let haveShimAMDSupport = 'amdModuleNames' in app;
      ^^^
Unexpected strict mode reserved word
D:\wamp64\www\__PROJECT__\node_modules\liquid-fire\index.js:83
      let haveShimAMDSupport = 'amdModuleNames' in app;
      ^^^
SyntaxError: Unexpected strict mode reserved word
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Function.Addon.lookup (D:\wamp64\www\__PROJECT__\node_modules\ember-cli\lib\models\addon.js:999:19)
    at D:\wamp64\www\__PROJECT__\node_modules\ember-cli\lib\models\addons-factory.js:47:38
    at visit (D:\wamp64\www\__PROJECT__\node_modules\ember-cli\lib\utilities\DAG.js:23:3)
```

Googling similar errors indicated that it might be down to outdated Node, NPM or Ember installs, so I've updated all three to the latest versions - but I still get the same error.

**Comment by ef4** [comment]
Older versions of npm don't support `let`. It was unintentional that this slipped in, we should still try to support those versions.

**Comment by DoctorOak** [comment]
Do you know if there is there a release that supports the latest Ember version that predates this change?

**Comment by ef4** [comment]
Fixed in 41743f41a33311a055a86b9b188d97d2332d13e7

**Comment by ef4** [comment]
Released in 0.26.3. Thanks for the bug report!
